### DISCUSSION TITLE
Can I use an organizational account without a real organization to become sponsored?

### DISCUSSION THREAD

**Discussion by gjaryczewski** [community/community#129438] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have created an organization account for a set of new projects, although I am the only one involved, and there is no "real" organization in law or fiscal terms. In my country (Poland), I have a status of self-employed, so I am also a legal firm, but that's the question: is it relevant to GitHub Sponsors' bylaws and regulations? Should I use a personal account or an organizational account? (I have a plan to build the organization shortly, but it can take some time.)

**Comment by TayabGhafor** [comment, marked as answer]
Certainly!

If you're self-employed in Poland and plan to build an organization later, using an organization account on GitHub is advisable. It separates your personal and professional projects clearly and prepares you for future growth and collaboration. This approach aligns with your current legal status and allows you to manage sponsorships more professionally. Make sure to consult with a tax advisor to understand any local implications related to receiving funds.

**Reply by gjaryczewski** [reply]
Thank your very much for this answer, everything is crystal clear, so let’s start playing!
